explainaboard/metrics/meta_evaluation.py

```diff
@@ -77,7 +78,9 @@ def calc_stats_from_data(
             return SimpleMetricStats(
                 np.array(
                     [
-                        corr_func(true, pred)[0]
+                        0
+                        if math.isnan(corr_func(true, pred)[0])
+                        else corr_func(true, pred)[0]
```

Comment:
This calls the same function twice. this checking needs to be wrapped by a function, or corr functions need to return always valid number.